packages/backend/src/open_payments/payment/outgoing/service.ts

```diff
@@ -252,8 +260,14 @@ async function validateGrant(
     return false
   }
 
-  //lock grant
-  await deps.grantReferenceService.lock(grant.grant, deps.knex)
+  // Lock grant
+  // TODO: update to use objection once it supports forNoKeyUpdate
```

Comment:
We have a PR that I _could_ include in the comment
- https://github.com/Vincit/objection.js/pull/2319

Reply:
I think it's ok for now. Also, I'm guessing reviews haven't been fast on this because of: https://github.com/Vincit/objection.js/issues/2335. Would be good to keep an eye on this thread